crates/red_knot_python_semantic/src/visibility_constraints.rs

```diff
@@ -285,6 +282,21 @@ impl ScopedVisibilityConstraintId {
     }
 }
 
+impl Idx for ScopedVisibilityConstraintId {
+    #[inline]
+    fn new(value: usize) -> Self {
+        assert!(value <= (SMALLEST_TERMINAL as usize));
```

Comment:
Does it make any difference if you remove the assert here just to make sure we compare apples with apples